src/components/Testimonial/index.jsx

```diff
@@ -0,0 +1,267 @@
+import React, { useState } from "react";
+import {
+    TextField,
+    Checkbox,
+    FormGroup,
+    FormControlLabel,
+    MenuItem,
+    Button,
+    CircularProgress,
+    Snackbar,
+    Card
+} from "@material-ui/core";
+import MuiAlert from "@material-ui/lab/Alert";
+import styles from "./testimonial.module.css";
+
+function Alert(props) {
+    return <MuiAlert elevation={6} variant="filled" {...props} />;
+  }
+
+const Testimonial = () => {
+  const [name, setName] = useState("");
+  const [studentNumber, setStudentNumber] = useState("");
+  const [program, setProgram] = useState("");
+  const [year, setYear] = useState("");
+  const [anonymousCheck, setAnonymousCheck] = useState(false);
+  const [comment, setComment] = useState("");
+  const [sending, setSending] = useState(false);
+  const [success, setSuccess] = useState(false);
+  const [failure, setFailure] = useState(false);
+  const [failText, setFailText] = useState("Something went wrong...");
+
+  const MAX_CHARS = 1500;
+
+  const enableSubmit = (
+    name !== "" &&
+    studentNumber !== "" &&
+    program !== "" &&
+    year !== "" &&
+    comment !== "" &&
+    comment.length <= MAX_CHARS &&
+    !sending
+  );
+
+  const handleAlertClose = (type) => (e, r) => {
+      if (r === 'clickaway') {
+          return;
+      }
+
+      type(false);
+      setFailText("Something went wrong...")
+  }
+
+  const handleYearChange = (event) => {
+    setYear(event.target.value);
+  };
+
+  const years = [
+    {
+      value: '1',
+      label: 'First',
+    },
+    {
+      value: '2',
+      label: 'Second',
+    },
+    {
+      value: '3',
+      label: 'Third',
+    },
+    {
+      value: '4',
+      label: 'Fourth',
+    },
+    {
+        value: '5',
+        label: 'Fifth+',
+    },
+    {
+        value: 'msc',
+        label: 'Masters',
+    },
+    {
+        value: 'phd',
+        label: 'Ph.D.',
+    }
+  ];
+
+  const handleClick = () => {
+      setSending(true);
+      const queryParams = {
+        headers: {
+            "Content-Type":"application/json; charset: UTF-8"
+        },
+        body: JSON.stringify({
+            name: name,
+            studentNumber: studentNumber,
+            program: program,
+            year: year,
+            comment: comment
+        }),
+        method: "POST"
+      }
+
+      const url = "/api/newAPIhere?";
```

Comment:
```suggestion
      // TODO: add new api here
      const url = "/api/newAPIhere?";
```

Will make it easier to track later